objects/src/notes/metadata.rs

```diff
@@ -47,6 +54,7 @@ impl From<&NoteMetadata> for Word {
         let mut elements = Word::default();
         elements[0] = metadata.tag;
         elements[1] = metadata.sender.into();
+        elements[2] = metadata.note_type().into();
         elements
```

Comment:
I think this approach is fine for now but if/when we add more items to the metadata, we may need to "compact" them a bit more so that we don't need more than a single word for it (having a single word allows us to hash note ID + metadata in a single permutation).

Generally, since metadata is always public we'll need to try to keep it as small as possible (at least for off-chain notes).

Reply:
Just to clarify, you have no request here, right?

Reply:
Yep - no requests, just general observation.